tailwind.config.js

```diff
@@ -4,5 +4,6 @@ module.exports = {
     theme: {
         extend: {},
     },
+    darkMode: ['class', '[data-theme="dark"]'],
```

Comment:
Do we even need `class`?

Reply:
https://tailwindcss.com/docs/dark-mode#customizing-the-class-name

Reply:
Hence my question. [`class` is a strategy that doesn't apply to us](https://tailwindcss.com/docs/dark-mode#customizing-the-class-name:~:text=If%20you%20want,the%20HTML%20tree.).

Reply:
Oh my bad, so i should change this to media?

Reply:
Wouldn't just `'[data-theme="dark"]'` work?

Reply:
The docs say it has to be the second item in the array

Reply:
> You can customize the dark mode selector name by setting darkMode to an array with your custom selector as the second item:

Reply:
Uhmmm, have you tried to see whether `class` is actually necessary? Cause I strongly suspect that it isn't.

Reply:
I haven't been able to yet, but you're probably rightm just depends on if they're doing [1] for that or just grabbing the last item of the array or something lol

Reply:
So, i just tested it. It does need to be like this.

Reply:
I tried with just the selector as well as with "media" instead of "class". Neither work.

btw, this is their typescript type 
![image](https://user-images.githubusercontent.com/34788790/221993845-c2fbbdde-2778-47d3-97f1-9e73bce9782c.png)

Reply:
Huh, so `class` is required. Weird, but okay.

Reply:
Sorry for the hold up then :)